scss/main.scss

```diff
@@ -7,5 +9,52 @@
 .container {
   display: flex;
   flex-direction: column;
-  padding: 71px 20px 0 20px;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: column;
+  padding: 71px 15px 38px 15px;
+
+  & .header-title {
+    color: $primary-color;
+    font-family: $font-tittle;
+    font-size: 36px;
+    font-weight: 700;
+    letter-spacing: -0.72px;
+    margin-bottom: 38px;
+  }
+
+  & .container-crew {
+    display: flex;
+    flex-direction: column;
+
+    & .team-title {
+      color: $primary-color;
+      font-family: $font-description;
```

Comment:
These values are exactly the same here and in `& .team-description`. Why don't you try to put those properties in a common selector or parent or whatever works for you?